contentcuration/contentcuration/frontend/shared/views/BottomBar.vue

```diff
@@ -17,9 +24,11 @@
   .bottom-bar {
     position: fixed;
     bottom: 0;
+    left: 0;
     z-index: 3;
+    display: flex;
     width: 100%;
-    height: 64px;
+    height: v-bind('appearanceOverrides.height');
```

Comment:
Lets have this a little more generic, instead of setting the specific property, we can have this css rule with 64px, but we can pass the `appearanceOverrides` object to the wrapper div like this:

```vue
  <div class="bottom-bar pa-2" :style="appearanceOverrides">
    <slot></slot>
  </div>
```
This way if the implementers wants to override any other property apart fromm the height can do it without having to update the component.